README.md

```diff
@@ -1,3 +1,45 @@
 # Fancy Website
 
 This is a fancy HTML website.
+
+## Here is a cat
+
+```
+
+          ,.                 .,
+         ,: ':.    .,.    .:' :,
+         ,',   '.:'   ':.'   ,',
+         : '.  '         '  .' :
+         ', : '           ' : ,'
+         '.' .,:,.   .,:,. '.'
+          ,:    V '. .' V    :,
+         ,:        / '        :,
```

Comment:
Not sure I like the nose.

Reply:
Not sure about the cat, rather a dog

Reply:
I'm not sure. I've had Agata's a dog biting my feet recently. Deffo in favour of cats right now.

Reply:
love the cat

Reply:
Ahh okay, I feel you. Whats your thoughts @sindatam

Reply:
Dog please.